web/src/compose_actions.js

```diff
@@ -240,13 +240,6 @@ export function start(msg_type, opts) {
         clear_box();
     }
 
-    if (msg_type === "stream") {
-        const $stream_header_colorblock = $(
-            "#compose_select_recipient_widget_wrapper .stream_header_colorblock",
-        );
-        stream_bar.decorate(opts.stream_id, $stream_header_colorblock);
-    }
-
```

Comment:
There is another reference to `stream_bar` in this file that needs to be removed